interview-questions.md

```diff
@@ -7,62 +7,63 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-  Your answer:
+  Your answer: A conditional statement is a sentence that implies something will happen if something is true or false; for instance if an animal can fly than it is a bird. In this sentence the conditions are animal and flying and if they are both true the sentence determines that it is a bird.
 
-  Researched answer:
+  Researched answer:  Conditionals are how decisions are framed in programming and they can perform different actions depending on true or false values.
 
 
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: git is a tool for source control developed by Linus Torvalds. Git allows you to create a repository ( a named folder ) that contains code with permissions for the repository owner to create branches for different code to be developed and then merged back with the main branch. 
 
+  Researched answer: Git is software for tracking changes. It is used to co ordinate among programmers. The main reasons to use git are speed, data integrity, and distributed working. 
```

Comment:
All researched answers need to be written **in your own words**!  Simply removing pieces from documentation and leaving parts of the definition left in tact will not be accepted.

Reply:
updated the answer